### PR TITLE
revision of exercises

### PR DESCRIPTION
**Pull request by ElviraQDP**

### CODE REVIEW

File: .learn/resets/04.1-Combined-Rules/styles.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.what {
+	width: 50px;
+	height: 50px;
+	padding-top: 10px;
```
Comment (alesanchezr) [inline .learn/resets/04.1-Combined-Rules/styles.css:4]: Can you please remove the `.learn/resets` folder? Before you commit on the exercises make sure the resets folders is deleted.

File: exercises/01.1-The-Style-Tag/README.es.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+tutorial: "https://www.youtube.com/watch?v=C5sOchuD2d4"
```
Comment (alesanchezr) [inline exercises/01.1-The-Style-Tag/README.es.md:2]: El tutorial no deberia estar porque esta hecho in ingles.

File: exercises/02-Separate-Stylesheet/README.es.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+tutorial: "https://www.youtube.com/watch?v=T0anCc5fvSg"
```
Comment (alesanchezr) [inline exercises/02-Separate-Stylesheet/README.es.md:2]: no esta hecho en ingles

File: exercises/03-Inline-Styles/README.es.md [hunk 4]
```diff
@@ -1,23 +1,23 @@
-# `03` Inline Styles (estilos alineados)
+# `03` Inline Styles (estilos en la línea)
 
-Inline styles son una muy mala idea a menos que no tengas otra opción (y esa es una situación muy poco común). Lamentablemente, tenemos que enseñarte cómo hacerlo porque existe la posibilidad de que necesites usarlos en algún momento.
+Los estilos inline son una muy mala idea a menos que no tengas otra opción (y esa es una situación muy poco común). Lamentablemente, tenemos que enseñarte cómo hacerlo porque existe la posibilidad de que necesites usarlos en algún momento.
 
-Para usar inline styles, en lugar de declarar una etiqueta <style> en el encabezado de el documento, debes establecer el atributo "estilo" de cualquier elemento con el código CSS que necesitas aplicar a ese elemento específico.
+Para usar estiloss inline, en lugar de declarar una etiqueta <style> en el header del documento, debes establecer el atributo "style" de cualquier elemento con el código CSS que necesitas aplicar a ese elemento específico.
```
Comment (alesanchezr) [inline exercises/03-Inline-Styles/README.es.md:5]: estilloss => estilos

File: exercises/12-Relative-Length-EM-REM/README.es.md [hunk 5]
```diff
@@ -1,19 +1,21 @@
 # `12` Longitud Relativa EM, REM
 
-Por ahora, estamos acostumbrados a trabajar con píxeles como unidad de medida, por ejemplo, diciendo que una caja tiene 100 px de ancho.
+HAsta ahora, hemos trabajado con píxeles como unidad de medida, por ejemplo, diciendo que una caja tiene 100 px de ancho.
```
Comment (alesanchezr) [inline exercises/12-Relative-Length-EM-REM/README.es.md:3]: hasta

**Review by alesanchezr — CHANGES_REQUESTED** [review]

File: .learn/resets/04.1-Combined-Rules/styles.css (hunk 1 above)
Comment (ElviraQDP) [reply .learn/resets/04.1-Combined-Rules/styles.css:4]: done

File: exercises/02-Separate-Stylesheet/README.es.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+# `02` Hoja de estilos separada
```
Comment (ElviraQDP) [inline exercises/02-Separate-Stylesheet/README.es.md:5]: done

**Review by alesanchezr — CHANGES_REQUESTED** [review]
can u remove the resets?

**Pull request merged**